UnitTestFrameworkPkg/Library/UnitTestLib/UnitTestLib.c

```diff
@@ -131,6 +131,61 @@ CompareFingerprints (
   return (CompareMem (FingerprintA, FingerprintB, UNIT_TEST_FINGERPRINT_SIZE) == 0);
 }
 
+STATIC
+EFI_STATUS
+FreeUnitTestTestEntry (
+  IN UNIT_TEST_LIST_ENTRY  *TestEntry
+  )
+{
+  // TODO: Finish this function.

```

Comment:
What's the goal in "finishing" the function?

Reply:
Old comment that needs to be removed now that the function is implemented

Reply:
Oh, that function is finished? Then the superfluous return type should be dropped, the result is discarded anyway.